Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -83,24 +83,67 @@ public extension MoyaProvider {
     }
 
     // swiftlint:disable:next function_parameter_count
-    private func performRequest(_ target: Target, request: URLRequest, callbackQueue: DispatchQueue?, progress: Moya.ProgressBlock?, completion: @escaping Moya.Completion, endpoint: Endpoint, stubBehavior: Moya.StubBehavior) -> Cancellable {
-        switch stubBehavior {
-        case .never:
-            switch endpoint.task {
-            case .requestPlain, .requestData, .requestJSONEncodable, .requestCustomJSONEncodable, .requestParameters, .requestCompositeData, .requestCompositeParameters:
-                return self.sendRequest(target, request: request, callbackQueue: callbackQueue, progress: progress, completion: completion)
-            case .uploadFile(let file):
-                return self.sendUploadFile(target, request: request, callbackQueue: callbackQueue, file: file, progress: progress, completion: completion)
-            case .uploadMultipart(let multipartBody), .uploadCompositeMultipart(let multipartBody, _):
-                guard !multipartBody.isEmpty && endpoint.method.supportsMultipart else {
+    private func performRequest(_ target: Target,
```

Comment:
Since most of the functions does not have a new line after each parameter, I would suggest to have the same here and in the following cases. Just to be consistent with the style.

Reply:
If functions have a lot of parameters like this I prefer breaking the style consistency to have it more readable.

Reply:
I partially agree with you, but consistency matters too. 
In case you would like to break after each parameter, apply this style in every function that has more than **x** parameters and open a pull request.

Reply:
That's a fair point. I've updated it.